pkg/controller/util.go

```diff
@@ -49,10 +49,11 @@ func (c *Controller) clusterWorkerID(clusterName spec.NamespacedName) uint32 {
 
 func (c *Controller) createOperatorCRD(crd *apiextv1beta1.CustomResourceDefinition) error {
 	if _, err := c.KubeClient.CustomResourceDefinitions().Create(crd); err != nil {
-		if !k8sutil.ResourceAlreadyExists(err) {
-			return fmt.Errorf("could not create customResourceDefinition: %v", err)
+		if k8sutil.ResourceAlreadyExists(err) {
+			c.logger.Infof("customResourceDefinition %q is already registered", crd.Name)
+		} else {
+			c.logger.Warnf("Can't register customResourceDefinition %s, will continue if it already exists", crd.Name)
```

Comment:
1. Please log the error for the ease of debugging as the original code does

Reply:
done